source/core/NumericString.cpp

```diff
@@ -1490,15 +1527,15 @@ Boolean TypedScanString(SubString* inputString, IntIndex* endToken, const Format
                             // reach the end of the input
                             i--;
                         }
-                        if (formatOptions->MinimumFieldWidth > 0 && i+1-stringStart > formatOptions->MinimumFieldWidth) {
+                        if (formatOptions->MinimumFieldWidth > 0 && i + 1 - stringStart > formatOptions->MinimumFieldWidth) {
```

Comment:
Please create another PR with the refactoring changes. We are going to cherry-pick this PR and create a Hot-fix for it. I want to cherry-pick the code that is strictly necessary.